lib/ansible/modules/network/nxos/nxos_file_copy.py

```diff
@@ -60,6 +67,36 @@
       - SSH port to connect to server during transfer of file
     default: 22
     version_added: "2.5"
+  file_pull:
+    description:
+      - Whether or not to pull the remote file from the device. If True,
```

Comment:
Suggest rewording as follows:
```
file_pull:
  description:
    - When (False) File is copied from the Ansible Server to the NXOS device.
    - When (True) File is copied from a remote SCP server to the NXOS device.
      In this mode, the file copy is initiated from the NXOS device.
    - If the file is already present on the device it will be overwritten and
      therefore the operation is NOT idempotent.